content/en/blog/r-udf/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Adding R UDF to Texera: The Journey"
+description: ""
+lead: "In this blog, we discuss the journey taken to implement support for R in Texera. Most notably, we allowed support for Table and Tuple API with R in Texera
+."
+date: 2024-07-06T08:00:00+00:00
```

Comment:
update the date to current date and time

Reply:
Fixed